packages/fxa-content-server/app/tests/spec/views/sms_send.js

```diff
@@ -158,6 +158,12 @@ describe('views/sms_send', () => {
         assert.lengthOf(view.$('.success'), 0);
       });
     });
+
+    it('renders `not now` links', () => {
+      return view.render().then(() => {
+        assert.lengthOf(view.$('.btn-not-now'), 1);
```

Comment:
@shane-tomlinson if I do a `click()` on this element, it will redirect current window to the link. However, that will make all the next mocha tests fail since the page moved.

Is there a better way to test this? Maybe this test isn't needed?

Reply:
I'd just migrate this whole thing to the functional tests. Or stub out `navigateAway` before clicking.

Reply:
Maybe check the link is displayed in the functional tests but don't click it. We've had recent issues about tests failing because of external dependencies and minimizing those would be worthwhile.

Reply:
Done!